credentials/build.gradle.kts

```diff
@@ -20,7 +20,11 @@ dependencies {
   api("decentralized-identity:did-common-java:1.9.0")
   api("decentralized-identity:uni-resolver-core:0.13.0")
 
+  implementation(project(":dids"))
+  implementation(project(":common"))
+  implementation(project(":crypto"))
   implementation("com.github.richardbergquist:java-multicodec:main-SNAPSHOT")
```

Comment:
we can remove this because it's no longer being used

Reply:
done https://github.com/TBD54566975/web5-kt/commit/8410886ad866254e6bb7cfe84285c00e904526e7